package/gluon-status-page-mesh-batman-adv/Makefile

```diff
@@ -0,0 +1,23 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-status-page-mesh-batman-adv
+PKG_VERSION:=1
+
+include ../gluon.mk
+
+define Package/gluon-status-page-mesh-batman-adv
+  TITLE:=Batman-data provider for gluon-status-page
+  SECTION:=gluon
+  CATEGORY:=Gluon
+  PROVIDES:=gluon-status-page-data-provider
```

Comment:
I think this PROVIDES is somewhat useless; as long as *gluon-status-page-mesh-batman-adv* depends on *gluon-status-page*, the result will be a circular dependency, which should always be avoided.

As the status page does not have a public API anymore, the status-page frontend for a specific mesh protocol will always be provided by the same package the backend comes in. As such, there is no common functionality in different backend packages that would justify a PROVIDES.